<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Creator</class>
 <widget class="QDialog" name="Creator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="data">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>551</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>data</string>
   </property>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="timer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>35</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="format">
     <string>%v/%m</string>
    </property>
   </widget>
   <widget class="QLabel" name="question">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>531</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Question</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="ansA">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="ansB">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="ansC">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="ansD">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QLabel" name="dLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>D</string>
    </property>
   </widget>
   <widget class="QLabel" name="aLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="bLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QLabel" name="cLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="ansN">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string>%v</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>none</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="leaderboard">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>551</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>leaderboard</string>
   </property>
   <widget class="QPushButton" name="exit_">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="leaderBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>531</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="leader">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>leader</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
